talk/owt/sdk/conference/conferenceclient.cc

```diff
@@ -1,20 +1,20 @@
 // Copyright (C) <2018> Intel Corporation
 //
 // SPDX-License-Identifier: Apache-2.0
-#include "talk/owt/sdk/include/cpp/owt/conference/conferenceclient.h"
 #include <algorithm>
 #include <string>
 #include "talk/owt/sdk/base/mediautils.h"
 #include "talk/owt/sdk/base/stringutils.h"
 #include "talk/owt/sdk/conference/conferencepeerconnectionchannel.h"
-#include "talk/owt/sdk/include/cpp/owt/base/stream.h"
 #include "talk/owt/sdk/include/cpp/owt/conference/remotemixedstream.h"
 #include "webrtc/api/stats_types.h"
 #include "webrtc/api/task_queue/default_task_queue_factory.h"
 #include "webrtc/rtc_base/critical_section.h"
 #include "webrtc/rtc_base/logging.h"
 #include "webrtc/rtc_base/task_queue.h"
 #include "webrtc/rtc_base/third_party/base64/base64.h"
+#include "talk/owt/sdk/include/cpp/owt/base/stream.h"
```

Comment:
why you move it to the end?

Reply:
to avoid the redefine of Status in x11 header files, have resolved this issue.